pages/my-profile/index.js

```diff
@@ -0,0 +1,59 @@
+import useSWR from "swr";
+//import Link from "next/link";
+import styled from "styled-components";
+import Image from "next/image";
+import Flex from "@/components/Layout/Flex";
+
+const StyledUserImage = styled(Image)`
+  border-radius: 50%;
+`;
+
+const StyledUserName = styled.div`
+  width: 260px;
+  height: 40px;
```

Comment:
I strongly recommend to use a relative unit here instead of fixed pixel values. Otherwise, this might not always look as you expected, think of very long usernames for example.